dashdblocal_notebooks/livy-notebooks/IoT_Demo_Pyspark.ipynb

```diff
@@ -0,0 +1,1246 @@
+    "\n",
+    "# -> *to cut out ?*\n",
+    "Run the following cell to launch the just deployed IoT landing app inside dashDB. The cell keeps a database connection open that we will use to stop the landing app again when you execute the subsequent cell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "FloatProgress(value=0.0, bar_style='info', description='Progress:', layout=Layout(height='25px', width='50%'),…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "('42724', '[42724] [IBM][CLI Driver][DB2/LINUXX8664] SQL0444N  Routine \"*RKSUBMIT\" (specific name \"SQL190923111753010\") is implemented with code in library or path \".../sqllib/function/SPARKSUBMIT\", function \"*SUBMIT\" which cannot be accessed.  Reason code: \"4\".  SQLSTATE=42724\\n')\n",
+      "Traceback (most recent call last):\n",
+      "  File \"/mnt/blumeta0/home/bluadmin/.local/lib/python3.7/site-packages/ibmdbpy/base.py\", line 762, in ida_query\n",
+      "    return sql.ida_query(self, query, silent, first_row_only, autocommit)\n",
+      "  File \"/mnt/blumeta0/home/bluadmin/.local/lib/python3.7/site-packages/ibmdbpy/sql.py\", line 81, in ida_query\n",
+      "    return _ida_query_ODBC(idadb, query, silent, first_row_only, autocommit)\n",
+      "  File \"/mnt/blumeta0/home/bluadmin/.local/lib/python3.7/site-packages/ibmdbpy/sql.py\", line 94, in _ida_query_ODBC\n",
+      "    cursor.execute(query)\n",
+      "  File \"/mnt/blumeta0/home/bluadmin/.local/lib/python3.7/site-packages/pypyodbc.py\", line 1626, in execute\n",
+      "    self.execdirect(query_string)\n",
+      "  File \"/mnt/blumeta0/home/bluadmin/.local/lib/python3.7/site-packages/pypyodbc.py\", line 1652, in execdirect\n",
+      "    check_success(self, ret)\n",
+      "  File \"/mnt/blumeta0/home/bluadmin/.local/lib/python3.7/site-packages/pypyodbc.py\", line 1007, in check_success\n",
+      "    ctrl_err(SQL_HANDLE_STMT, ODBC_obj.stmt_h, ret, ODBC_obj.ansi)\n",
+      "  File \"/mnt/blumeta0/home/bluadmin/.local/lib/python3.7/site-packages/pypyodbc.py\", line 975, in ctrl_err\n",
+      "    raise ProgrammingError(state,err_text)\n",
+      "pypyodbc.ProgrammingError: ('42724', '[42724] [IBM][CLI Driver][DB2/LINUXX8664] SQL0444N  Routine \"*RKSUBMIT\" (specific name \"SQL190923111753010\") is implemented with code in library or path \".../sqllib/function/SPARKSUBMIT\", function \"*SUBMIT\" which cannot be accessed.  Reason code: \"4\".  SQLSTATE=42724\\n')\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "import ibmdbpy\n",
+    "from ibmdbpy import IdaDataBase,IdaDataFrame, IdaGeoDataFrame\n",
```

Comment:
nitpicking: you can remove the imports of `IdaDataFrame` and `IdaGeoDataFrame`, because you never use them! :-)